mycqu/card.py

```diff
@@ -149,25 +149,36 @@ class EnergyFees:
     """
     balance: float
     """账户余额"""
-    electricity_subsidy: float
-    """电剩余补助"""
-    water_subsidy: float
-    """水剩余补助"""
+    electricity_subsidy: Optional[float]
+    """电剩余补助（仅虎溪校区拥有）"""
+    water_subsidy: Optional[float]
+    """水剩余补助（仅虎溪校区拥有）"""
+    subsidies: Optional[float]
+    """补助余额（仅老校区拥有）"""
 
     @staticmethod
-    def from_dict(data: dict[str, Any]) -> EnergyFees:
+    def from_dict(data: dict[str, Any], isHuXi: bool) -> EnergyFees:
```

Comment:
参数 `isHuxi` 请改成 snake case 命名法 `is_huxi`
```suggestion
    def from_dict(data: dict[str, Any], is_huxi: bool) -> EnergyFees:
```